packages/components/Checkbox/src/CheckboxTokens.mobile.ts

```diff
@@ -0,0 +1,48 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { CheckboxTokens } from './Checkbox.types';
+import { Platform } from 'react-native';
+
+// No Tokens for Sizes , hovered and circular checkbox as they're not supported for Android Platform
```

Comment:
Should we change comment to be "not supported on mobile platforms"

Reply:
Yes, good catch!
I was actually thinking of splitting this into `.android` and `.ios` because these controls actually have very different specs on Android/iOS.

Reply:
I ended up splitting Checkbox and Radio back to separate files altogether.  @ankraj / @rohanprasadofficial FYI.